### ISSUE TITLE
No streams match your pattern

### ISSUE THREAD

**Issue by sergeyarl** [jorgebastida/awslogs#148]
Hi! 

Can you please check if this is a normal behaviour: 

```
$ awslogs streams -s "30m ago" ECS-logs
ecs-testapp01/ecs-testapp01/4dde9959-e098-4167-9730-b0555f44bc9d
ecs-testapp01/ecs-testapp01/ad223d1e-cebe-4abb-9806-8e6762af9b1c

$ awslogs get -w ECS-logs ecs-testapp01/*
No streams match your pattern 'ecs-testapp01/*'.

$ awslogs get ECS-logs ecs-testapp01/*
No streams match your pattern 'ecs-testapp01/*'.

$ awslogs get -w ECS-logs ecs-testapp01/ecs-testapp01/ad223d1e-cebe-4abb-9806-8e6762af9b1c
No streams match your pattern 'ecs-testapp01/ecs-testapp01/ad223d1e-cebe-4abb-9806-8e6762af9b1c'.

$ awslogs get -w ECS-logs ecs-testapp01/ecs-testapp01/4dde9959-e098-4167-9730-b0555f44bc9d
No streams match your pattern 'ecs-testapp01/ecs-testapp01/4dde9959-e098-4167-9730-b0555f44bc9d'.

```
Thanks.

**Comment by sergeyarl** [comment]
After some time it started to work. 

`$ awslogs get -w ECS-logs ecs-testapp01/*` works just fine now. So this issues occurs from time to time.

**Comment by sergeyarl** [comment]
And now it isn't working again :)

**Comment by Martijn02** [comment]
Try using the same start parameter (-s) as you used to get your streams. 

It's caused by a bug/feature in AWS where the last updated time for a stream can be more than an hour behind. If you don't specify a -s, it will use 5m as a default. It won't look into the streams listed in your streams command, because, according to AWS, there are no new log entries in them.

**Comment by jorgebastida** [comment]
https://github.com/jorgebastida/awslogs/pull/199 should fix this issue. These changes will be released as part of ``0.11.0``.
